### ISSUE TITLE
apostrophe's still don't export right

### ISSUE THREAD

**Issue by csnewman88** [haffonist/igamt#1175]
It looks like we still have the issue with apostrophe's not always exporting properly. The original ticket (1102) is closed but I don't see a resolution in it.

![image](https://cloud.githubusercontent.com/assets/17408249/22471374/c568432c-e797-11e6-9ba1-884be3036a12.png)

![image](https://cloud.githubusercontent.com/assets/17408249/22471528/35419a4a-e798-11e6-9403-e402da824691.png)

**Comment by haffonist** [comment]
This seems to be caused by a copy/paste from another word document. 
Please you click on the icon </> and paste the content ?

**Comment by csnewman88** [comment]
<p>This field contains the patient&#39;s sex.&nbsp;</p>

<p>This field refers to the patient&#39;s race.&nbsp;</p>

**Comment by haffonist** [comment]
Here are the actual data:
```
This field contains the patient&apos;s sex.&nbsp;
This field contains the patient&apos;s date and time of birth. 
```
So it is not the export's issue but instead a data issue. My guess is this comes from a word document. 
Normally copying and pasting in an editor before pasting back to IGAMT should fix it. Currently that will be a solution until we figure out how to remove special character automatically.

**Comment by csnewman88** [comment]
OK. I fixed them in my IG. they are exporting OK now.

**Comment by maxence-lefort** [comment]
Should be fixed with the pull request #1199
